racket/collects/racket/contract/private/arr-i.rkt

```diff
@@ -1066,22 +1126,27 @@ evaluted left-to-right.)
                          [i (in-naturals)])
                 (string->symbol (format "post-proc~a" i)))
 
+            ;; the param procs
```

Comment:
I'm not sure this comment is helping 😄 . The comments in the ones below seem like they provide some value to me, but this one really just repeats what's two lines below.